README.md

```diff
@@ -62,11 +62,14 @@ Histogram can be used to visualize parameter distribution and trends for any ten
 </p>
 
 ## Quick Start
+To give the VisualDL a quick test, please use the following commands.
```

Comment:
So instead of this, why don't we just give them a simple command "virtualenv venv; source venv/bin/activate`. This requires them to go read another doc and figure out how virtualenv works. What do you think?

Reply:
Yeah just saw your stuff for contrib. I'd just add those lines here too and avoid the confusiuon.